client/blocks/inline-help/popover.jsx

```diff
@@ -168,18 +198,51 @@ class InlineHelpPopover extends Component {
 }
 
 function mapStateToProps( state ) {
+	const siteId = getSelectedSiteId( state );
+	const currentRoute = getCurrentRoute( state );
+	const classicRoute = currentRoute
```

Comment:
Since we're only jumping from Gutenberg URLs Classic editor ones, why not simply something like this?
```es6
const classicRoute = currentRoute.replace( '/gutenberg/', '/' );
```

Reply:
Good call, I'm not sure why I went with the whole band of Array methods but this is definitely more succinct. Updated!

Noting that I passed an empty string vs a `/` for the replacement string. Starting the URL path with a `/` omits the host in the redirect.